docs/source/get_started/new_project.md

```diff
@@ -132,6 +132,10 @@ Now run the project:
 kedro run
 ```
 
+```{warning}
+`kedro run` requires at least one pipeline with nodes. Please define a pipeline before running this command.
```

Comment:
I think this might be helpful for those using Kedro first time:

```suggestion
`kedro run` requires at least one pipeline with nodes. Please define a pipeline before running this command, and ensure it is registred in `pipeline_registry.py`.
```